src/sgx/rust_sgxffi/src/lib.rs

```diff
@@ -0,0 +1,111 @@
+
```

Comment:
Copyright header missing.

Reply:
Sure, will take care of updating copyright